### ISSUE TITLE
Multiple kubeconfig files broken

### ISSUE THREAD

**Issue by dwradcliffe** [Shopify/kubernetes-deploy#463]
# Bug report

Since https://github.com/Shopify/kubernetes-deploy/pull/428, if you have KUBECONFIG with multiple files, kubernetes-deploy does not work at all.

The `--kubeconfig` flag on kubectl only supports a single file, where the env var supports merging multiple files.

**Expected behavior:**
No error.

**Actual behavior:**

```
[INFO][2019-04-06 10:57:05 -0400]	------------------------------------Phase 1: Initializing deploy------------------------------------
[DEBUG][2019-04-06 10:57:05 -0400]	Running command (attempt 1): kubectl config get-contexts -o name --kubeconfig=/Users/davidradcliffe/.kube/config:/Users/davidradcliffe/.kube/config1:/Users/davidradcliffe/.kube/config2 --request-timeout=15
[DEBUG][2019-04-06 10:57:05 -0400]	Kubectl out:
[INFO][2019-04-06 10:57:05 -0400]
[INFO][2019-04-06 10:57:05 -0400]	------------------------------------------Result: FAILURE-------------------------------------------
[FATAL][2019-04-06 10:57:05 -0400]	Context foo is not available. valid contexts: []
```

**Version(s) affected:**
v0.26.2

### Steps to Reproduce

1. `export KUBECONFIG=~/.kube/config1:~/.kube/config2`
2. `kubernetes-deploy foo bar`

**Comment by benlangfeld** [comment]
I propose that we revert #428.

https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#merging-kubeconfig-files

@KnVerey thoughts?

**Comment by dturn** [comment]
Yep, that's definitely the problem. I've got an idea for how to fix forward and I'd rather do that than revert #428.  I'll try and push up a branch tomorrow.

**Comment by dwradcliffe** [comment]
👍 Fixed in `0.26.3`
